src/Test/L0/Worker/WorkerCommandManagerL0.cs

```diff
@@ -14,9 +15,13 @@ public TestWorkerCommandExtensionL0()
                 CommandArea = "TestL0";
                 SupportedHostTypes = HostTypes.All;
                 InstallWorkerCommand(new FooCommand());
-                InstallWorkerCommand(new Foo2Command());
                 InstallWorkerCommand(new BarCommand());
             }
+
+            public void InstallFoo2Command()
+            {
+                InstallWorkerCommand(new FooCommand());
```

Comment:
My understanding is this is set up so we throw when installing more than one of the same command.

Would be more clear if we rename this maybe something like `InstallSecondFooCommand`

Reply:
actually, that was a typo. I meant to install a Foo2Command